introduction.md

```diff
@@ -1,2 +1,5 @@
 Hi, I'm Dewsmen and I'm doing GitHub cource on codeschool.com 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?